lib/openpgp.js

```diff
@@ -1,23 +0,0 @@
-/* START.NODE_ONLY */
-global.btoa = require('btoa');
-global.atob = require('atob');
-const openpgp = require('openpgp');
-/* END.NODE_ONLY */
-
-/* START.BROWSER_ONLY */
-/**
- * @link https://github.com/vibornoff/asmcrypto.js/issues/121
- */
-asmCrypto.random.skipSystemRNGWarning = true;
-
-const { hardwareConcurrency = 1 } = window.navigator || {};
-openpgp.initWorker({ path: 'openpgp.worker.min.js', n: hardwareConcurrency });
-/* END.BROWSER_ONLY */
-
-openpgp.config.integrity_protect = true;
```

Comment:
I don't see a reason to move this to the app itself. I think we should have a global setup controlled in this app. That would allow us to handle defaults that are changing in the main library more easily.

Reply:
Agreed :)

After discussion with @KAYLukas and @sanjanarajan we decided to leave the configuration in pmcrypto but the worker initialization inside the user of the library because the path to the worker will change depending on the browser, and nodejs does not support workers, and maybe (I can't give a concrete example) we don't want to have the worker in some app or in some browser. This just lets us be more flexible.

Reply:
We need to use a function to pass the path to pmcrypto, no ?

Reply:
I mean I'm fine to add a helper function to create a worker in openpgp to reduce the duplication of two lines, but I think it should not automatically create one.

pmcrypto is just a library of helper functions, it should not do magic in my opinion.